tests/threshold/threshold-config-threshold-threshold-flow/threshold.config

```diff
@@ -0,0 +1 @@
+threshold gen_id 1, sig_id 1000001, type threshold, track by_flow, count 5, seconds 60
```

Comment:
Test name `threshold-config-threshold-threshold-flow` has a majority of `threshold` words :-p

Could some comment state that 19 // 5 == 3 (+ remainder 4 which did not reach the threshold yet)